server/routes/orderhistory.js

```diff
@@ -0,0 +1,29 @@
+import express from "express";
+import cors from 'cors';
+import db from "../db/db-connection.js";
+const router = express.Router();
+
+
+//GET REQUEST FOR ORDER HISTORY
```

Comment:
+1 for comments!

For API endpoints sometimes showing an example JSON output is helpful at a glance too for what data the client can expect  back.